apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloAutoPersistedQueryInterceptor.java

```diff
@@ -20,15 +20,27 @@
 
   private final ApolloLogger logger;
   private volatile boolean disposed;
+  private boolean useHttpGetMethodForQueries;
```

Comment:
I guess this flag can be local in `interceptAsync `?

Reply:
removed